diff --git a/node_modules/emoji-picker-react/dist/emoji-picker-react.cjs.development.js b/node_modules/emoji-picker-react/dist/emoji-picker-react.cjs.development.js
index 23f65cf..b332e58 100644
--- a/node_modules/emoji-picker-react/dist/emoji-picker-react.cjs.development.js
+++ b/node_modules/emoji-picker-react/dist/emoji-picker-react.cjs.development.js
@@ -302,34 +302,34 @@ var SuggestedRecent = {
 };
 var configByCategory = (_configByCategory = {}, _configByCategory[exports.Categories.SUGGESTED] = {
   category: exports.Categories.SUGGESTED,
-  name: 'Frequently Used'
+  name: '최근 사용'
 }, _configByCategory[exports.Categories.CUSTOM] = {
   category: exports.Categories.CUSTOM,
-  name: 'Custom Emojis'
+  name: '사용자 정의'
 }, _configByCategory[exports.Categories.SMILEYS_PEOPLE] = {
   category: exports.Categories.SMILEYS_PEOPLE,
-  name: 'Smileys & People'
+  name: '스마일리 및 사람'
 }, _configByCategory[exports.Categories.ANIMALS_NATURE] = {
   category: exports.Categories.ANIMALS_NATURE,
-  name: 'Animals & Nature'
+  name: '동물 및 자연'
 }, _configByCategory[exports.Categories.FOOD_DRINK] = {
   category: exports.Categories.FOOD_DRINK,
-  name: 'Food & Drink'
+  name: '음식 및 음료'
 }, _configByCategory[exports.Categories.TRAVEL_PLACES] = {
   category: exports.Categories.TRAVEL_PLACES,
-  name: 'Travel & Places'
+  name: '여행 및 장소'
 }, _configByCategory[exports.Categories.ACTIVITIES] = {
   category: exports.Categories.ACTIVITIES,
-  name: 'Activities'
+  name: '활동'
 }, _configByCategory[exports.Categories.OBJECTS] = {
   category: exports.Categories.OBJECTS,
-  name: 'Objects'
+  name: '사물'
 }, _configByCategory[exports.Categories.SYMBOLS] = {
   category: exports.Categories.SYMBOLS,
-  name: 'Symbols'
+  name: '기호'
 }, _configByCategory[exports.Categories.FLAGS] = {
   category: exports.Categories.FLAGS,
-  name: 'Flags'
+  name: '깃발'
 }, _configByCategory);
 function baseCategoriesConfig(modifiers) {
   return categoriesOrdered.map(function (category) {
diff --git a/node_modules/emoji-picker-react/dist/emoji-picker-react.esm.js b/node_modules/emoji-picker-react/dist/emoji-picker-react.esm.js
index 264f3d5..bfdc58d 100644
--- a/node_modules/emoji-picker-react/dist/emoji-picker-react.esm.js
+++ b/node_modules/emoji-picker-react/dist/emoji-picker-react.esm.js
@@ -1,4 +1,4 @@
-import React__default, { useRef, createElement, createContext, createRef, useContext, useState, useEffect, useMemo, useCallback, Fragment, memo, Component } from 'react';
+import React__default, { Component, Fragment, createContext, createElement, createRef, memo, useCallback, useContext, useEffect, useMemo, useRef, useState } from 'react';
 
 var ClassNames;
 
@@ -302,39 +302,39 @@ var SkinTonePickerLocation;
 var _configByCategory;
 var categoriesOrdered = [Categories.SUGGESTED, Categories.CUSTOM, Categories.SMILEYS_PEOPLE, Categories.ANIMALS_NATURE, Categories.FOOD_DRINK, Categories.TRAVEL_PLACES, Categories.ACTIVITIES, Categories.OBJECTS, Categories.SYMBOLS, Categories.FLAGS];
 var SuggestedRecent = {
-  name: 'Recently Used',
+  name: '최근 사용한 이모지',
   category: Categories.SUGGESTED
 };
 var configByCategory = (_configByCategory = {}, _configByCategory[Categories.SUGGESTED] = {
   category: Categories.SUGGESTED,
-  name: 'Frequently Used'
+  name: '최근 사용한 이모지'
 }, _configByCategory[Categories.CUSTOM] = {
   category: Categories.CUSTOM,
-  name: 'Custom Emojis'
+  name: '사용자 정의'
 }, _configByCategory[Categories.SMILEYS_PEOPLE] = {
   category: Categories.SMILEYS_PEOPLE,
-  name: 'Smileys & People'
+  name: '스마일리 및 사람'
 }, _configByCategory[Categories.ANIMALS_NATURE] = {
   category: Categories.ANIMALS_NATURE,
-  name: 'Animals & Nature'
+  name: '동물 및 자연'
 }, _configByCategory[Categories.FOOD_DRINK] = {
   category: Categories.FOOD_DRINK,
-  name: 'Food & Drink'
+  name: '음식 및 음료'
 }, _configByCategory[Categories.TRAVEL_PLACES] = {
   category: Categories.TRAVEL_PLACES,
-  name: 'Travel & Places'
+  name: '여행 및 장소'
 }, _configByCategory[Categories.ACTIVITIES] = {
   category: Categories.ACTIVITIES,
-  name: 'Activities'
+  name: '활동'
 }, _configByCategory[Categories.OBJECTS] = {
   category: Categories.OBJECTS,
-  name: 'Objects'
+  name: '사물'
 }, _configByCategory[Categories.SYMBOLS] = {
   category: Categories.SYMBOLS,
-  name: 'Symbols'
+  name: '기호'
 }, _configByCategory[Categories.FLAGS] = {
   category: Categories.FLAGS,
-  name: 'Flags'
+  name: '깃발'
 }, _configByCategory);
 function baseCategoriesConfig(modifiers) {
   return categoriesOrdered.map(function (category) {
diff --git a/node_modules/emoji-picker-react/src/components/footer/Preview.css b/node_modules/emoji-picker-react/src/components/footer/Preview.css
index 37a9b41..d9bae01 100644
--- a/node_modules/emoji-picker-react/src/components/footer/Preview.css
+++ b/node_modules/emoji-picker-react/src/components/footer/Preview.css
@@ -6,6 +6,7 @@
   height: var(--epr-preview-height);
   position: relative;
   z-index: var(--epr-preview-z-index);
+  display: none;
 }
 
 .EmojiPickerReact .epr-preview .epr-preview-emoji-label {
diff --git a/node_modules/emoji-picker-react/src/config/categoryConfig.ts b/node_modules/emoji-picker-react/src/config/categoryConfig.ts
index e3a60aa..5f8f9a6 100644
--- a/node_modules/emoji-picker-react/src/config/categoryConfig.ts
+++ b/node_modules/emoji-picker-react/src/config/categoryConfig.ts
@@ -28,43 +28,43 @@ export type CustomCategoryConfig = {
 const configByCategory: Record<Categories, CategoryConfig> = {
   [Categories.SUGGESTED]: {
     category: Categories.SUGGESTED,
-    name: 'Frequently Used'
+    name: '최근 사용한 이모지'
   },
   [Categories.CUSTOM]: {
     category: Categories.CUSTOM,
-    name: 'Custom Emojis'
+    name: '사용자 정의'
   },
   [Categories.SMILEYS_PEOPLE]: {
     category: Categories.SMILEYS_PEOPLE,
-    name: 'Smileys & People'
+    name: '스마일리 및 사람'
   },
   [Categories.ANIMALS_NATURE]: {
     category: Categories.ANIMALS_NATURE,
-    name: 'Animals & Nature'
+    name: '동물 및 자연'
   },
   [Categories.FOOD_DRINK]: {
     category: Categories.FOOD_DRINK,
-    name: 'Food & Drink'
+    name: '음식 및 음료'
   },
   [Categories.TRAVEL_PLACES]: {
     category: Categories.TRAVEL_PLACES,
-    name: 'Travel & Places'
+    name: '여행 및 장소'
   },
   [Categories.ACTIVITIES]: {
     category: Categories.ACTIVITIES,
-    name: 'Activities'
+    name: '활동'
   },
   [Categories.OBJECTS]: {
     category: Categories.OBJECTS,
-    name: 'Objects'
+    name: '사물'
   },
   [Categories.SYMBOLS]: {
     category: Categories.SYMBOLS,
-    name: 'Symbols'
+    name: '기호'
   },
   [Categories.FLAGS]: {
     category: Categories.FLAGS,
-    name: 'Flags'
+    name: '깃발'
   }
 };
 
